lib/formfields/admin/domains/formfield.domains_edit.php

```diff
@@ -390,6 +390,13 @@
 						'value' => '1',
 						'checked' => $result['openbasedir']
 					],
+					'openbasedir_path' => [
+						'visible' => $result['openbasedir'] == '1',
```

Comment:
what if the value for `openbasedir` is to be changed when editing and then the admin can only see this option when entity is saved and reopened

Reply:
True, it's better when the field is always visible